feat(icons): Add toilet icon

EthanHazel opened the pull request:
<!-- Thank you for contributing! -->

<!-- Insert `closes #issueNumber` here if merging this PR will resolve an existing issue -->
Closes #1788

## What is the purpose of this pull request?
<!-- Please choose one of the following, and put an "x" next to it. -->
- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description
<!-- Please insert your description here and provide info about the "what" this PR is contribution -->
Creates a toilet icon SVG and JSON.

### Icon use case <!-- ONLY for new icons, remove this part if not icon PR -->
<!-- What is the purpose of this icon? For each icon added, please insert at least two real life use cases (the more the better). Text like "it's a car icon" is not accepted. -->
Can be used for things such as house / hotel listings to show the amount of bathrooms available.
Health related apps that could use it for medical related reasonings.
Maps showing you where the nearest bathroom is located.

## Icon Design Checklist <!-- ONLY for new icons, remove this part if not icon PR -->

### Concept <!-- ONLY for new icons -->
<!-- All of these requirements must be fulfilled. -->
- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license<!-- ONLY for new icons. -->
<!-- Please choose one of the following, and put an "x" next to it. -->
- [x] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [ ] I've based them on the following Lucide icons: <!-- provide the list of icons -->
- [ ] I've based them on the following design: <!-- provide source URL and license permitting use -->

### Naming <!-- ONLY for new icons -->
<!-- All of these requirements must be fulfilled. -->
- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design <!-- ONLY for new icons -->
<!-- All of these requirements must be fulfilled. -->
- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting <!-- For every PR! -->
<!-- All of these requirements must be fulfilled. -->
- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

ericfennis commented on icons/toilet.svg:
```diff
@@ -0,0 +1,13 @@
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M7,12h13s1,0,1,1c0,4-4,5-7,5,0,0,2,2,2,3s-1,1-1,1H6s-1,0-1-1,2-3,2-3c0,0-4-1-4-5V4c0-1,1-2,2-2H13c1,0,2,1,2,2V12"/>
```
Can you format this code?
Comas should be replaced with spaces

gorgobacka commented:
@EthanHazel Are you going to fix this?

@ericfennis Is the linting the only point that needs to be adjusted? And would it be possible to re-submit this PR using the same icon, if nothing happens in a reasonable time?  I would like to see this icon in another app that is using this package.

EthanHazel commented:
@ericfennis Just fixed the formatting of the SVG, sorry for the late reply

jguddas commented:
More arcs:
<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M7+12h13a1+1+0+0+1+1+1+5+5+0+0+1-5+5h-.598a.5.5+0+0+0-.424.765l1.544+2.47a.5.5+0+0+1-.424.765H5.402a.5.5+0+0+1-.424-.765L7+18%22+/%3E%3Cpath+d=%22M8+18a5+5+0+0+1-5-5V4a2+2+0+0+1+2-2h8a2+2+0+0+1+2+2v8%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNNyAxMmgxM2ExIDEgMCAwIDEgMSAxIDUgNSAwIDAgMS01IDVoLS41OThhLjUuNSAwIDAgMC0uNDI0Ljc2NWwxLjU0NCAyLjQ3YS41LjUgMCAwIDEtLjQyNC43NjVINS40MDJhLjUuNSAwIDAgMS0uNDI0LS43NjVMNyAxOCIgLz4KICA8cGF0aCBkPSJNOCAxOGE1IDUgMCAwIDEtNS01VjRhMiAyIDAgMCAxIDItMmg4YTIgMiAwIDAgMSAyIDJ2OCIgLz4KPC9zdmc+.svg"/><br/>Open lucide studio</a>

EthanHazel commented:
> More arcs: [<img alt="icons" width="200px" src="https://camo.githubusercontent.com/d841196f717803ee6a770114724008acfcb1363e4c9547751dd3ea97e8c30fa8/68747470733a2f2f6c75636964652e6465762f6170692f67682d69636f6e2f50484e325a776f67494868746247357a50534a6f644852774f693876643364334c6e637a4c6d39795a7938794d4441774c334e325a79494b4943423361575230614430694d6a516943694167614756705a326830505349794e43494b4943423261575633516d393450534977494441674d6a51674d6a5169436941675a6d6c7362443069626d39755a53494b4943427a64484a7661325539496d4e31636e4a6c626e5244623278766369494b4943427a64484a766132557464326c6b64476739496a496943694167633352796232746c4c577870626d566a59584139496e4a766457356b49676f6749484e30636d39725a5331736157356c616d3970626a3069636d3931626d5169436a344b49434138634746306143426b50534a4e4e7941784d6d67784d324578494445674d434177494445674d534178494455674e534177494441674d5330314944566f4c5334314f5468684c6a55754e534177494441674d4330754e4449304c6a63324e5777784c6a55304e4341794c6a5133595334314c6a55674d434177494445744c6a51794e4334334e6a56494e5334304d444a684c6a55754e534177494441674d5330754e4449304c5334334e6a564d4e7941784f4349674c7a344b49434138634746306143426b50534a4e4f4341784f474531494455674d434177494445744e533031566a52684d694179494441674d434178494449744d6d6734595449674d694177494441674d53417949444a324f4349674c7a344b5043397a646d632b2e737667">Open lucide studio](https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M7+12h13a1+1+0+0+1+1+1+5+5+0+0+1-5+5h-.598a.5.5+0+0+0-.424.765l1.544+2.47a.5.5+0+0+1-.424.765H5.402a.5.5+0+0+1-.424-.765L7+18%22+/%3E%3Cpath+d=%22M8+18a5+5+0+0+1-5-5V4a2+2+0+0+1+2-2h8a2+2+0+0+1+2+2v8%22+/%3E)

Looks great! Added this version to the PR and added you to contributors

EthanHazel reviewed (commented):
Fixed issues related to naming and tags

ericfennis commented:
@EthanHazel Can you check the linter?

EthanHazel commented:
> @EthanHazel Can you check the linter?

Just ran the prettier script and should pass fine now

ericfennis reviewed (approved):
Nice looks good now

EthanHazel reviewed (commented):
test
